backend/src/schema/resolvers/rewards.spec.js

```diff
@@ -191,58 +253,61 @@ describe('rewards', () => {
 
     describe('check test setup', () => {
       it('user has one badge', async () => {
-        const query = gql`
+        authenticatedUser = regularUser.toJson()
+        const userQuery = gql`
           {
-            User(id: "u1") {
+            User(id: "regular-user-id") {
               badgesCount
               badges {
                 id
               }
             }
           }
         `
-        const expected = { User: [{ badgesCount: 1, badges: [{ id: 'indiegogo_en_rhino' }] }] }
-        const client = new GraphQLClient(host)
-        await expect(client.request(query)).resolves.toEqual(expected)
+        const expected = {
+          data: { User: [{ badgesCount: 1, badges: [{ id: 'indiegogo_en_rhino' }] }] },
+          errors: undefined,
+        }
+        await expect(query({ query: userQuery })).resolves.toMatchObject(expected)
       })
     })
 
     describe('unauthenticated', () => {
-      let client
-
       it('throws authorization error', async () => {
-        client = new GraphQLClient(host)
-        await expect(client.request(mutation, variables)).rejects.toThrow('Not Authorised')
+        authenticatedUser = null
+        await expect(mutate({ mutation, variables })).resolves.toMatchObject({
```

Comment:
don't be afraid to follow your instincts and be prescriptive with your variable naming...